concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -160,6 +167,25 @@ jobs:
     plan:
       - get: paas-cf
         trigger: {{auto_deploy}}
+      - get: concourse-manifest
+      - task: self-update-pipeline
+        config:
+          platform: linux
+          image: docker:///governmentpaas/self-update-pipelines
+          inputs:
+            - name: paas-cf
+            - name: concourse-manifest
+          params:
+            DEPLOY_ENV: {{deploy_env}}
+            BRANCH: {{branch_name}}
+            MAKEFILE_ENV_TARGET: {{makefile_env_target}}
+            SELF_UPDATE_PIPELINE: {{self_update_pipeline}}
+            PIPELINES_TO_UPDATE: {{pipeline_name}}
+          run:
+            path: bash
+            args:
+              - ./paas-cf/concourse/scripts/self-update-pipeline.sh
```

Comment:
Any reason to run this via bash here as opposed to just calling the script directly?